src/utils/stack.js

```diff
@@ -0,0 +1,14 @@
+import Contentstack from 'contentstack';
+
```

Comment:
Although I just re-factored the gql queries using apolloClient, we may want to refactor it again using CS-Delivery SDK to query data now that we have contentstack installed...

Reply:
Sigh should have thought about this... but here is the [ticket](https://jira.mongodb.org/browse/DEVHUB-1831)